### PR TITLE
Propose test freeze deadline for 1.20

### PR DESCRIPTION
**Pull request by jeremyrickard**
#### What type of PR is this:

/kind documentation
/area release-team
/milestone v1.20
/priority critical-urgent

#### What this PR does / why we need it:

This PR proposes a *Test Freeze* date in Week 11.  Previously, the schedule omitted a test freeze deadline. 

/hold for discussion

Signed-off-by: Jeremy Rickard <rickardj@vmware.com>

### CODE REVIEW

**Comment by spiffxp** [comment]
/lgtm
From my end as SIG Testing chair and Conformance subproject owner, but I would like to see input from at least @kubernetes/ci-signal

File: releases/release-1.20/README.md [hunk 1]
```diff
@@ -71,6 +72,7 @@ The 1.20 release cycle is proposed as follows:
 | **KubeCon CloudNativeCon North America Starts** | Community | Tue, November 17 |  | |
 | **KubeCon CloudNativeCon North America Ends** | Community | Friday, November 20 |  | |
 | Docs deadline - PRs ready for review | Docs Lead | Mon November 23 | week 11 | |
+| **[Test Freeze]** (EOD PST) | Branch Manager | Mon November 23 | |
```
Comment (tpepper) [inline releases/release-1.20/README.md:75]: Down a couple lines "Code Thaw" ought to become "Code and Test Thaw"?  Or just "Thaw" and link to release_phases.md#thaw with that getting yet another clarification from it's current "Code Thaw" being generic thawing for frozen stuff like code and test.

**Review by tpepper — CHANGES_REQUESTED** [review]

File: releases/release-1.20/README.md (hunk 1 above)
Comment (jeremyrickard) [reply releases/release-1.20/README.md:75]: OK, I can update the phases document. I think a more generic `Thaw` makes sense:

**Review by hasheddan — APPROVED** [review]
/lgtm

From dates perspective. @RobertKielty could you take a peek here as you'll have the reins this go around :)

File: releases/release-1.20/README.md (hunk 1 above)
Comment (tpepper) [reply releases/release-1.20/README.md:75]: This text looks reasonable to me.

Comment (jeremyrickard) [reply releases/release-1.20/README.md:75]: @tpepper take a look at the updated phases document please.

Comment (jeremyrickard) [reply releases/release-1.20/README.md:75]: Actually, need to fix the references in handbooks.

**Review by tpepper — APPROVED** [review]
/lgtm

**Pull request merged**